### ISSUE TITLE
Add guidance for joining community call in low-bandwidth environnments

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3538]
### Summary

@richarddushime very kindly pointed out that our Zoom calls are not accessible for people calling in from low-bandwidth environments. 

On our public-facing pads, we have links to the Zoom - but we don't have information about how people can join using other tools: like calling in from their personal phones (an option that is possible on the Zoom mobile app, but not clear)

### What needs to be done?

- [ ] Add documentation to the guides - either in the community handbook or the Accessibility Guide to help people learn how they can join the call using the tools they have available to them.
- [ ] Add information about joining the calls from mobile to the respective pads (Collab cafe and/or coworking call)

### Who can help?

* Anyone interested in documenting practices that can help make our resource more accessible in low-bandwidth environments!

**Comment by richarddushime** [comment]
It seems like my internet connection is causing disruptions. It's a common issue, possibly due to the instability of my internet connection. Next time, I'll consider using a phone call instead, as suggested.

**Comment by richarddushime** [comment]
It seems like my internet connection is causing disruptions. It's a common issue, possibly due to the instability of my internet connection. Next time, I'll consider using a phone call instead, as suggested.

**Comment by aleesteele** [comment]
That's a great piece of advice, @richarddushime!

**Comment by aleesteele** [comment]
Closing this for now
